runtime/near-vm-logic/Cargo.toml

```diff
@@ -31,7 +31,8 @@ near-vm-errors = { path = "../near-vm-errors", version = "3.0.0" }
 bn = { package = "zeropool-bn", version = "0.5.9", features = [], optional = true }
 
 [dev-dependencies]
-serde_json = {version= "1", features= ["preserve_order"]}
+serde_json = { version = "1", features = ["preserve_order"] }
+serde_with = { version = "^1.9", features = ["hex"] }
```

Comment:
I personally would avoid adding this dependency. I don't feel strongly about this (-0), so I am fine with having it if you prefer. 

Here's the rational (wordy not because I hate this, but because I want to make sure we are on the same page regarding the tradeoffs):

* this is a new depency for the project, it isn't already present in Cargo.lock (new deps increase compile time, number of maintainers we need to trust, risks that `cargo update` fails due to someone messing up semver)
* this dependency is used only in tests, and in a single test (the benefit we get out of it is small in scope)
* this is "convenience" dependency -- it doesn't contain any meaningful computational logic, and just helps to make the syntax nicer (it tradeoffs amount of code you need to write for API surface you need to understand, which is zero-sumish). 
* this is a "proc macro on proc macro" dependency, and proc macros are inherently brittly and poorly composable part of the Rust language (so, its hard to understand how this thing actually works). 

What I'd do instead is define 

```rust
#[derive(Deserialize)]
struct EcrecoverTest {
    m: String,
    v: u8,
    sig: String,
    mc: bool,
    res: Option<String>,
}
```

and then use the `hex` library (which is already present in Cargo.lock, and which encapsulate the actual decode hex computation logic) in the test to manually decode the free fields. Alternatively, I'd use `https://serde.rs/field-attrs.html#deserialize_with` directly.